packages/bus-iterator-transform/lib/ActorIteratorTransform.ts

```
@@ -0,0 +1,174 @@
+ * @see IActionIteratorTransform
+ * @see IActorIteratorTransformOutput
+ */
+export abstract class ActorIteratorTransform<TS = undefined>
+  extends Actor<ActionIteratorTransform, IActorTest, ActorIteratorTransformOutput, TS> {
+  public wraps: possibleOperationTypes[];
+  /**
+   * @param args - @defaultNested {<default_bus> a <cc:components/Bus.jsonld#Bus>} bus
+   */
+  public constructor(args: IActorIteratorTransformArgs<TS>) {
+    super(args);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)